clients/gateway/gateway.go

```diff
@@ -77,6 +77,14 @@ func NewClient(gatewayURL string, log utils.SimpleLogger) *Client {
 }
 
 func (c *Client) AddInvokeTransaction(txn json.RawMessage) (json.RawMessage, error) {
+	return c.addTransaction(txn)
+}
+
+func (c *Client) AddDeployAccountTransaction(txn json.RawMessage) (json.RawMessage, error) {
+	return c.addTransaction(txn)
```

Comment:
Since the gateway only exposes the `add_transaction` endpoint, can we just have one `AddTransaction` function?

Reply:
Yes, it would require the methods to have the same input and output parameters, but it would make the code neater. I'll update the code to reflect this.